docs/troubleshooting.md

```diff
@@ -0,0 +1,35 @@
+# Troubleshooting an installed system
+
+
+## Using `toolbox` for debugging
+
+Operating Systems deployed by Elemenal are image based and immutable during
+runtime. Any change would be applied only after reboot. That solution has several benefits, but it also has some disadvantages. If you need to debug your system and install a new tool, the tool will be available only after reboot. Therefore, you cannot debug the currently running system. For this reason, a utility called toolbox has been developed.
```

Comment:
> Any change would be applied only after reboot.

> If you need to debug your system and install a new tool, the tool will be available only after reboot.

Is this copied from Micro docs? None of it applies to Elemental.

Reply:
Removed.